x/mongo/driver/batch_cursor.go

```diff
@@ -140,6 +144,7 @@ type CursorOptions struct {
 	CommandMonitor *event.CommandMonitor
 	Crypt          Crypt
 	ServerAPI      *ServerAPIOptions
+	Registry       *bsoncodec.Registry
```

Comment:
Add "Registry" to the CursorOptions struct. This will be used to set the unexpected "registry" field on the driver.BatchCursor struct.

The idea is to allow to internally pass user-defined registries to the batch cursor, for encoding comments.